js_modules/dagster-ui/packages/ui-core/src/assets/overview/ComputeDetailsSection.tsx

```diff
@@ -38,6 +40,16 @@ export const ComputeDetailsSection = ({
         </Tag>
       </AttributeAndValue>
 
+      <AttributeAndValue label={pools.length > 1 ? 'Pools' : 'Pool'}>
+        {pools.length > 0 ? (
+          <Box flex={{gap: 4}}>
+            {pools.map((pool, idx) => (
+              <PoolTag key={idx} pool={pool} />
+            ))}
+          </Box>
+        ) : null}
```

Comment:
Should there be any kind of content for the `0` case?

Reply:
I don't think so...

Reply:
Maybe `None` or something? Or not show the section?

Reply:
I think the `AttributeAndValue` already does this (hides the section)?